nixos/modules/programs/traceroute.nix

```diff
@@ -0,0 +1,26 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.programs.traceroute;
+in {
+  options = {
+    programs.traceroute = {
+      enable = mkOption {
+        type = types.bool;
+        default = false;
+        description = ''
+          Whether to configure a setcap wrapper for traceroute.
+        '';
+      };
+    };
+  };
+
+  config = mkIf cfg.enable {
+    security.wrappers.traceroute = {
```

Comment:
Should probably also do `environment.systemPackages = [ pkgs.traceroute ]`

Reply:
what for?
to add `${traceroute}/bin` to $PATH ?
there is nothing but `traceroute` executable which is shadowed by `/run/wrappers/bin/traceroute`.

Reply:
Ah I see, never mind then :)